theme/views.py

```diff
@@ -114,12 +114,12 @@ def get_context_data(self, **kwargs):
 class UserPasswordResetView(TemplateView):
     template_name = 'accounts/reset_password.html'
 
-    def get_context_data(self, **kwargs):
-        token = kwargs.pop('token', None)
```

Comment:
The token is authenticated in the call before the redirect here so I removed since the token wasn't even checked here.

Reply:
@sblack-usu I did see the check below: 
```
        if token is None:
            raise ValidationError('Unauthorised access to reset password')
```
Most likely this simple None check is not needed since token is already checked before redirecting, but bring this up just in case.

Reply:
yeah, I removed it because it wasn't doing anything.